src/test/java/com/katelocate/menugenerator/RecipeRepositoryTests.java

```diff
@@ -164,4 +164,22 @@ void shouldSaveAll() {
 
         assertThat(jdbcTemplate.query("SELECT * FROM Recipe", new RecipeRowMapper())).isEqualTo(testRecipes);
     }
+
+    @Test
+    void shouldFindByType() {
```

Comment:
I don't see that you add any recipe to the repo prior to the test, while before each test you delete all the recipes from the repo. Is it intentional?

Reply:
Sorry, I'll fix it... I missed that no test objects were saved to postgres.